embedJson/gen/builder-aws-ami.json

```diff
@@ -12,6 +12,7 @@
          "Default": "",
          "EnvVar": "",
          "Category": false,
+         "Example": "",
```

Comment:
I'm not really sure why anything in `embedJson` changed when I regenerated website mdx, but it seems benign.

Reply:
Ah, I think this is because of the `waypoint-plugin-sdk` changes. It added a new `Example` field, and so the re-gen is adding these for every doc given that SDK change.